src/components/LogInForm/LogInForm.js

```diff
@@ -1,42 +1,44 @@
-import React from 'react';
+import SubmitButton from "common/buttons/SubmitButton.js";
+import FormContainer from "common/forms/FormContainer";
+import PasswordInput from "common/forms/PasswordInput";
+import TextInput from "common/forms/TextInput";
+import useField from "hooks/useField";
 import { useDispatch } from "react-redux";
-import { useNavigate } from 'react-router-dom';
-import { FormContainer, PasswordInput, SubmitButton, TextInput } from '../../formElements';
-import useField from '../../hooks/useField';
-import { logUserIn } from '../../reducers/loggedInUserReducer';
-import loginService from '../../services/login';
+import { useNavigate } from "react-router-dom";
+import { logUserIn } from "reducers/loggedInUserReducer";
+import loginService from "services/login";
 
 const LogInForm = () => {
   const navigate = useNavigate();
-  const name = useField('text');
-  const password = useField('password');
+  const name = useField("text");
```

Comment:
Use react-hook-form and yup for forms. It will be easier to coordinate and validate fields.
It is a bit tricky to start so go through docs and medium articles.
https://react-hook-form.com/
https://github.com/jquense/yup
https://dev.to/franciscomendes10866/react-form-validation-with-react-hook-form-and-yup-4a98

Open a new JIRA ticket and do this task with that. Do not address it here.